src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -206,6 +223,7 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
       <Panel
```

Comment:
Not quite following you on this one?  The panel is in it's own component right now.  Unless you mean to split it so that the panel content and the panel itself are separated so the content is just a child component of the panel itself?